src/V3Builtin.cpp

```diff
@@ -0,0 +1,90 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: built-in packages and classes
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2004-2020 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#include "V3Ast.h"
+#include "V3Builtin.h"
+#include "V3FileLine.h"
+#include "V3Parse.h"
+#include "V3ParseSym.h"
+
+namespace V3Builtin {
+void parseStdPackage(V3Parse& parser) {
+    string filename = v3Global.opt.hierTopDataDir() + "/builtin.v";
+    std::ofstream* ofp = nullptr;
+    ofp = V3File::new_ofstream(filename);
```

Comment:
```suggestion
    const std::unique_ptr<std::ofstream> ofp(V3File::new_ofstream(filename));
```

Much safer and then you can remove close and delete below.
Also if open fails you should error out.